src/Domain/Creators/CylindricalBinaryCompactObject.cpp

```diff
@@ -390,18 +396,14 @@ CylindricalBinaryCompactObject::CylindricalBinaryCompactObject(
   // spheres aren't included, the outer radii would have to be in the middle of
   // a block. With the inner spheres, the outer radii can be at block
   // boundaries.
-  if (not (include_inner_sphere_A and include_inner_sphere_B)) {
+  if (not(include_inner_sphere_A and include_inner_sphere_B)) {
     PARSE_ERROR(context,
                 "To use the CylindricalBBH domain with time-dependent maps, "
                 "you must include the inner spheres for both objects. "
                 "Currently, one or both objects is missing the inner spheres.");
   }
-  is_time_dependent_ = true;
-  initial_time_ = initial_time;
-  expansion_map_options_ = expansion_map_options;
-  initial_angular_velocity_ = initial_angular_velocity;
-  initial_size_map_values_[0] = initial_size_map_values_A;
-  initial_size_map_values_[1] = initial_size_map_values_B;
+
+  time_dependent_options_ = time_dependent_options;
```

Comment:
std::move this, or are you sure it's trivially copyable?

Reply:
Clang-tidy yelled at me when I originally had `std::move` and said it's trivially copyable. I was surprised by this